linchpin/__init__.py

```diff
@@ -122,6 +122,8 @@ def runcli(ctx, config, workspace, verbose, version, creds_path):
 
     if workspace is not None:
         ctx.workspace = os.path.realpath(os.path.expanduser(workspace))
+    else:
```

Comment:
I've been sitting here debating this option, I don't think this is the best place for this option.  After thinking and reviewing more, I see what I missed and what needs to happen. Essentially, the `linchpin init` functionality needs to be moved from linchpin/__init__.py to linchpin/cli/__init__.py. This will allow the workspace to be set correctly, as it inherits from the api/__init__.py. Let me get a fix in place, but I don't think I want to accept this fix.